OxfordBodleian/Juel-Jensen/BDLaethg38.xml

```diff
@@ -233,9 +233,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     </physDesc>
                     <history>
                         <origin>
-                            <origDate notBefore="1800" notAfter="2006" evidence="lettering"/>
+                            <origDate notBefore="1800" notAfter="2006" evidence="lettering">nineteenth–twenty-first century</origDate>
+                            The manuscript can be dated palaeographically to the nineteenth–twentieth century.
+                            The name of the orignal owner, mentioned in supplications throughout, was <persName ref="PRS13973WalattaMa"/>.
```

Comment:
Maybe "The original owner, …, was …"? This may be a more general thing that we can look into when reviewing the pdf.